lib/OpenQA/WebAPI/Controller/Test.pm

```diff
@@ -303,8 +303,10 @@ sub _show {
     for my $key (OpenQA::Schema::Result::Jobs::SCENARIO_WITH_MACHINE_KEYS) {
         push(@conds, {"me.$key" => $job->get_column($key)});
     }
-    my $limit_previous = $self->param('limit_previous') // 10;    # arbitrary limit of previous results to show
-    my %attrs = (
+    my $limit_previous = $self->param('limit_previous') // 10;                                                                                                                                                          # arbitrary limit of previous results to show
```

Comment:
Looks like the comment is in the right line but our tidy script likes spaces so the line is wrapped.

I'll move the comment above the current line though to work around the issue with tidy.